src/PowerShellEditorServices/Services/DebugAdapter/Handlers/ConfigurationDoneHandler.cs

```diff
@@ -121,6 +116,8 @@ private async Task LaunchScriptAsync(string scriptToLaunch)
                     ScriptBlockAst ast = Parser.ParseInput(untitledScript.Contents, untitledScript.DocumentUri.ToString(), out Token[] tokens, out ParseError[] errors);
 
                     // This seems to be the simplest way to invoke a script block (which contains breakpoint information) via the PowerShell API.
+                    //
+                    // TODO: Fix this so the added script doesn't show up.
                     var cmd = new PSCommand().AddScript(". $args[0]").AddArgument(ast.GetScriptBlock());
                     await _executionService
                         .ExecutePSCommandAsync<object>(cmd, CancellationToken.None, s_debuggerExecutionOptions)
```

Comment:
This is the cause of https://github.com/PowerShell/vscode-powershell/issues/3666

Reply:
@SeeminglyScience Also saw this.

Reply:
Yeah that'll do it for sure. I believe there's supposed to be some option on `ExecutePSCommandAsync` or it's options class that allows you to specify a "display/history" string for the command.